tonic_validate/utils/llm_calls.py

```diff
@@ -7,7 +7,7 @@
 
 
 def similarity_score_call(
-    question: str, reference_answer: str, llm_answer: str, openai_service: OpenAIService
+        question: str, reference_answer: str, llm_answer: str, openai_service: OpenAIService
```

Comment:
Minor nit, the spacing here doesn't match the rest of the file. It also doesn't pass the formatter we use (ruff). Can you undo the spacing changes here?

Reply:
Done.